tests/test_fare_attributes.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+const { expect } = require('chai');
+const { Gtfs } = require('../index');
+
+describe('Tests on GTFS fare attributes', () => {
+  it('Tests on gtfs.getNumberOfFareAttributes(), .add…(), .remove…() and .getFareAttributeWithId', (done) => {
```

Comment:
What happens if you add/remove a fare that doesn't exist?

Reply:
Removing a fare that doesn't exist won't do anything. It is like deleting an undefined key.

Adding a fare which doesn't exist should not work.